packages/lodestar-validator/src/services/block.ts

```diff
@@ -76,7 +78,7 @@ export default class BlockProposingService {
     const epoch = computeEpochAtSlot(this.config, slot);
     const randaoDomain = getDomain(
       this.config,
-      {fork} as BeaconState,
+      {fork, genesisValidatorsRoot} as BeaconState,
```

Comment:
Might be cool to finaly fix getDomain to require `Pick<BeaconState, "fork"|"genesisValidatorsRoot">` instead of whole BeaconState